code/src/main/java/com/google/sps/data/Business.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.data;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+
+@Entity
+public class Business {// This class is solely to check if a business was contacted in the last week.
+
+  @Id String placeId;
+  private long timeOfLastContactUnixTimeSeconds;
+
+  private Business() {}
```

Comment:
If this will never be called, we should remove it.

Reply:
Any class that is loaded into Objectify needs a private constructor

Reply:
Ah, TIL!